www/templates/jsontestresults.html

```diff
@@ -0,0 +1,206 @@
+{% extends "layout.html" %}
```

Comment:
Is it worthwhile sharing the template with the XML based test result as well? I.e Could extend a base file that they both use with overrides